models/Category.py

```diff
@@ -57,4 +58,16 @@ def budgeted_amount(self, amount):
     @property
     def prettytable_repr(self):
         self.activity_amount = self.budgeted_amount - self.available_amount
-        return [(self.id, self.name), self.budgeted_amount, -(self.activity_amount), self.available_amount]
+
+        if self.available_amount == 0:

```

Comment:
Myślę, że można to zrefaktorować - zapisać "style" do zmiennej i umieścić go w if-ach, a nie "availabe_amount" jak obecnie.